src/pbkdf2.rs

```diff
@@ -133,12 +133,14 @@ use {constant_time, digest, hmac, polyfill};
 ///
 /// `derive` panics if `iterations < 1`.
 ///
-/// `derive` panics if `out.len()` is larger than the output length of the
-/// digest function used by the PRF algorithm.
+/// `derive` panics if `out.len()` is larger than 2^32 - 1, per the
```

Comment:
Please use the Python notation `2**32`. As I noted below, the spec actually says something slightly different, so we need to change this somehow to not lie.